core/domain/feedback_services.py

```diff
@@ -178,6 +183,19 @@ def create_message(
     new_status = thread.status
     thread.put()
 
+    # We do a put on the suggestion linked to the thread, if any so that the
+    # last_updated time changes to show that there is activity in the thread.
```

Comment:
How would registering an activity on the thread help?

Reply:
So my idea was that we accept suggestions which have no activity on them. By activity I mean any conversation basically. So if someone messages on a feedback thread linked to a suggestion, I want to update the last updated time of the suggestion too. Does this clarify the question?

Reply:
Adding on... If people are discussing about a suggestion, we don't want to auto-approve it right? If no one is discussing, and no action is being taken, then we can safely assume no activity is there and then we can accept the suggestion.

Reply:
Yep, that makes sense. Thanks!